rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -513,10 +506,14 @@ void trophy_manager_dialog::ResizeGameIcons()
 	if (m_game_combo->count() <= 0)
 		return;
 
+	QList<int> indices;
```

Comment:
Ditto

Reply:
i is used as well as compared as an int in the following code. so size_t is not making any sense here